flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/ScalingMetricEvaluator.java

```diff
@@ -137,6 +138,12 @@ private Map<ScalingMetric, EvaluatedScalingMetric> evaluateMetrics(
                         latestVertexMetrics.get(TRUE_PROCESSING_RATE),
                         getAverage(TRUE_PROCESSING_RATE, vertex, metricsHistory)));
 
+        evaluatedMetrics.put(
+                LOAD_MAX,
```

Comment:
I would suggest exposing this as `LOAD` and let this work with both LOAD_MAX and LOAD_AVG scaling metric. The default is already to use LOAD_MAX.

Reply:
Hi @mxm I chatted with @gyfora briefly on this and implemented a single LOAD metric instead. I've pushed the changes, have a look at it pls.

Reply:
Now we can visualise the business the same way operator uses it.